src/strings/en-gb.json

```diff
@@ -207,6 +207,8 @@
     "EnableBackdropsHelp": "Display the backdrops in the background of some pages while browsing the library.",
     "EnableCinemaMode": "Cinema mode",
     "EnableDisplayMirroring": "Display mirroring",
+    "EnableDisplayRatingsInPlayer": "Display ratings during playback",
```

Comment:
Don't manually edit languages other than `en-us` - this may cause conflicts with Weblate. Use Weblate for translation. Also, I don't think we need to translate GB if it is the same in US.